### ISSUE TITLE
Is the widening factor of S2GAN's feature extractor really 16?

### ISSUE THREAD

**Issue by RuiLiFeng** [google/compare_gan#35]
Hi, I've read your paper "High-Fidelity Image GenerationWith Fewer Labels". It's  a very fascinating work but I have one question about the pretrained feature extractor F.
![image](https://user-images.githubusercontent.com/33709183/66270387-80295e80-e885-11e9-8a4e-a31306c9ce9a.png)

In the paper you say you use ResNet50 V2 with widening factor 16, which is abnormally large, including  4 times more parameters than the BigGAN discriminator. My computer even cannot successfully construct it in CPU.

My resnet v2 code is from pytorch official repo https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py. I wonder whether I am misunderstanding your paper.

**Comment by Marvin182** [comment]
Yes, we used ResNet50 V2 with a widening factor of 16. We are working on open sourcing the code that as well.

**Comment by RuiLiFeng** [comment]
@Marvin182 Thank you for your reply, that helps me a lot. Look for work to your feature works!

**Comment by RuiLiFeng** [comment]
@Marvin182 By the way, did you have experiments with smaller widening factor, such as 2 or 4, could that outperform the co-training version?

**Comment by Marvin182** [comment]
No, we did not experiment with smaller widening factors.
